src/NuGet.config

```diff
@@ -1,16 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Changed during integration somehow. I'm copy/pasting from https://github.com/dotnet/corefx/blob/master/src/NuGet.config in my latest iteration.